src/main/java/Thames/Thames.java

```diff
@@ -0,0 +1,59 @@
+package Thames;
+
+import java.io.FileNotFoundException;
+
+import Thames.command.Command;
```

Comment:
Do you think we need to explicitly import all
classes, or is importing the package sufficient?